pkg/config/config_test.go

```diff
@@ -135,6 +135,29 @@ unknown_key.unknown_subkey: true
 	assert.Len(t, findUnknownKeys(confWithUnknownKeys), 0)
 }
 
+func TestUnknownVarsWarning(t *testing.T) {
+	test := func(v string, unknown bool) func(*testing.T) {
+		return func(t *testing.T) {
+			os.Setenv(v, "foo")
+			defer func() {
+				os.Unsetenv(v)
+			}()
+			var exp []string
+			if unknown {
+				exp = append(exp, v)
+			}
+			assert.Equal(t, exp, findUnknownEnvVars(Mock()))
+		}
+	}
+	t.Run("DD_API_KEY", test("DD_API_KEY", false))
+	t.Run("DD_SITE", test("DD_SITE", false))
+	t.Run("DD_UNKOWN", test("DD_UNKOWN", true))
```

Comment:
I think the linter is right on this one
```suggestion
	t.Run("DD_UNKNOWN", test("DD_UNKNOWN", true))
```